addons/project_purchase/models/project_project.py

```diff
@@ -115,26 +115,23 @@ def _get_profitability_items(self, with_action=True):
             purchase_order_lines = self.env['purchase.order.line'].sudo().search_fetch([
                 ('state', 'in', ['purchase', 'done']),
                 ('analytic_distribution', 'in', self.analytic_account_id.ids),
+                ('qty_invoiced', '>', 0),
```

Comment:
We could review the domain used for `search_fetch`, I think we can just keep the 3 first tuples inside that domain.

What do you think? 🙂

Reply:
Yes I think we can safely remove the other tuples as the specs have changed, I've tested this way it seems to work :)